### ISSUE TITLE
Ubuntu 20.04 - Cannot install Wok - Package 'python-m2crypto' has no installation candidate

### ISSUE THREAD

**Issue by wolffstarr** [kimchi-project/wok#300]
When attempting to install the listed dependencies for Wok on the wiki for install instructions under Ubuntu 20.04, I am getting the following error:

`wolffstarr@nike:~$ sudo apt install systemd logrotate python3-psutil python3-ldap python3-lxml python3-websockify python3-jsonschema openssl nginx python3-cherrypy3 python3-chee
tah python3-pam python-m2crypto gettext python3-openssl
Reading package lists... Done
Building dependency tree
Reading state information... Done
Package python-m2crypto is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'python-m2crypto' has no installation candidate
wolffstarr@nike:~$`

I have installed python3-m2crypto, but it is obviously not meeting the requirements. This is a fresh install of the Ubuntu 20.04 beta Live Server installer. Only additional changes are to install Cockpit Project. If it matters (can't see why it would) the platform is a Ryzen 5 1600 in an Asrock Rack X470D4U server board.

**Comment by NZLostboy** [comment]
Having the same issue, definitely not hardware related. Probably due to Ubuntu removing python2 stuff for version 20.

**Comment by jsandorski** [comment]
I worked around this issue by manually editing the dependencies of the deb package. This helped me a lot:
https://forums.linuxmint.com/viewtopic.php?t=35136

Changed python-m2crypto to python3-m2crypto

I do not see the virtualization tab in wok yet, though. Working on it ...

**Comment by NZLostboy** [comment]
Yeah saw that, I built from source and Wok starts up and runs but can't see the virtualization tab either. Haven't had time to go back to it

**Comment by Omemanti** [comment]
> I worked around this issue by manually editing the dependencies of the deb package. This helped me a lot:
> https://forums.linuxmint.com/viewtopic.php?t=35136
> 
> Changed python-m2crypto to python3-m2crypto
> 
> I do not see the virtualization tab in wok yet, though. Working on it ...

lifesaver :)

that worked like a charm. and indeed, the virt tab. is still missing.
